src/app/features/observations/cells/VerificationCell.tsx

```diff
@@ -9,38 +9,77 @@ import {
 } from "@material-ui/icons";
 import React from "react";
 import { connect, DispatchProp } from "react-redux";
-import { TmpObservation } from "../../../../store/reducers/observationListReducer";
+import { setObservationVerificationStatus } from "../../../../store/actions/observationListActions";
+import { ObservationData, VerificationStatus } from "../models";
 
 export const VerificationCell = connect()(
   ({
     dispatch,
     observation
-  }: DispatchProp & { observation: TmpObservation }) => (
-    // TODO update api and wire up
+  }: DispatchProp & { observation: ObservationData }) => (
     <>
-      {observation.verified ? (
-        <IconButton className="p-0 mr-2" disableRipple>
-          <RadioButtonUnchecked />
-        </IconButton>
-      ) : (
+      {observation.verificationStatus === "Pending" ? (
```

Comment:
Could we lift this check on to of the function as a variable?
`const isPending = observation.verificationStatus === "Pending"` 
Could be done also for Reject and Verified.